docs/usage.md

```diff
@@ -193,18 +233,18 @@ A location does not have a prescribed shape. In your UI it can represent a singl
 
 The location property will be set on the [member](#members).
 
-Because locations are part of members, a `memberUpdate` event will be emitted when a member updates their location. When a member leaves, their location is set to `null`.
+Because locations are part of members, a member event will be emitted when a member updates their location. When a member leaves, their location is set to `null`.
```

Comment:
I'm not sure this is true any longer. Location is part of a member, however "a member event" makes it sound like it would be picked up by `members.subscribe()`. AFAIK it won't be, it would only be picked up by `space.subscribe()`.